tests/ClientTest.php

```diff
@@ -265,6 +266,41 @@ public function testGetLastEventId()
         $this->assertEquals('ddbd643a51904ccea6ce3098506f9d33', $client->getLastEventID());
     }
 
+    public function testGetUserContext()
```

Comment:
Maybe we can collapse all those tests into one?

Reply:
We could, but I think it's much more clear if there is a test for each function